src/providers/device/__mocks__/device.mock.ts

```diff
@@ -0,0 +1,31 @@
+import { IDeviceProvider } from '../device.model';
+import { Injectable } from '@angular/core';
+// import { Device } from '@ionic-native/device';
```

Comment:
Couple of left-over comments in this file.

Reply:
Comments removed.